docker.go

```diff
@@ -328,6 +332,7 @@ func commandBuild(build Build) *exec.Cmd {
 		}
 	}
 
+	os.Setenv("DOCKER_BUILDKIT", "1")
```

Comment:
wondering if we should add a `buildkit: true` parameter to the yaml?  I'm not too familiar with the impact of enabling this by default.  Are there any potential consequences we should be aware of?